app/upgrade/upgrade_handler.go

```diff
@@ -0,0 +1,111 @@
+package lupercalia
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+)
+
+var addressesToBeAdjusted = []string{
+	"juno1aeh8gqu9wr4u8ev6edlgfq03rcy6v5twfn0ja8",
+}
+
+//BurnCoinFromAccount undelegate all amount in whale address (bypass 28 day unbonding), and send it to dead address
+func BurnCoinFromAccount(ctx sdk.Context, accAddr sdk.AccAddress, staking *stakingkeeper.Keeper, bank *bankkeeper.BaseKeeper) {
+	bondDenom := staking.BondDenom(ctx)
```

Comment:
I think we need something like this to also complete active redelegations... 

I'll see if I can test this locally:

```suggestion
	bondDenom := staking.BondDenom(ctx)
	
	another_now := ctx.BlockHeader().Time
	
		// this loop will complete all delegator's active redelegations
	for _, activeRedelegation:= range staking.GetRedelegations(ctx, accAddr) {
		// validator address of this unbonding delegation
		validatorSrcStringAddr := activeRedelegation.ValidatorSrcAddress
        	validatorDstStringAddr := activeRedelegation.ValidatorDstAddress
		validatorSrcValAddr, _ := sdk.ValAddressFromBech32(validatorSrcStringAddr)
		validatorDstValAddr, _ := sdk.ValAddressFromBech32(validatorDstStringAddr)

		// set all entry completionTime to now so we can complete redelegation
		for i := range activeRedelegation.Entries {
			activeRedelegation.Entries[i].CompletionTime = another_now 
		}
		staking.SetRedelegation(ctx, activeRedelegation)
		_, err := staking.CompleteRedelegation(ctx, accAddr, validatorSrcValAddr, validatorDstValAddr)
		if err != nil {
			panic(err)
		}
	}
```